packages/core/src/utils/is.ts

```diff
@@ -45,9 +47,9 @@ export const isPrimitive = ((value) => !isObject(value) && !isFunction(value)) a
   string | number | bigint | boolean | symbol
 >;
 
-export const isEventDefined = !isUndefined(Event);
+export const isEventDefined = !isUndefined(globalObject?.Event);
```

Comment:
Couldn't get a polyfill of Event to work, so instead just adding some error protection. Otherwise in the React Native environment you would get a "ReferenceError.... Cannot find variable: Event"